test/Libraries/RevitIntegrationTests/GlobalParameterTests.cs

```diff
@@ -17,7 +17,7 @@ namespace RevitSystemTests
     class GlobalParameterTests : RevitSystemTestBase
     {
         [Test]
-        [TestModel(@".\empty-2017.rvt")]
```

Comment:
This was probably done to avoid version migrations during test run across different versions.